src/components/ui/Tag/README.md

```diff
@@ -0,0 +1,172 @@
+| New | New, updated, or beta. | New, trial, information, help, alpha, beta. |
+| InProgress | In progress or current. | In progress, open, modified. |
+| Success | Completed or successful. | Available, done, approved, resolved, added, complete. |
+Danger | Failure or deletion. | Errors, declined, failed, removed, critical. |
+Warning | Attention or hazard. | Busy, blocked, missing, warning, information.
+
+Colour mappings can be customised in your theme.
+
+### Theming
+
+Generally speaking, the styling of Tags should be controlled in your theme. Tags should look the same across your site so that users can immediately recognise their purpose.
+
+Capitalisation, colour placement (background or border), border-radius, and padding are all customisable via your theme.
+
+## Behaviour
+
+### A tag can have an associated action
+
+A Tag can act as a link or a button, depending on context. Clicking on the tag should direct the user to more information about that tag. For instance, clicking the tag may take the user to a page listing only posts with that tag, or it may filter the current list to show/hide content using the tag.
+
+### A tag can be removed
+
+You may want to use Tags to allow users to add and remove attributes from an object. Do this by passing an event function to the Tag's `onDismiss` prop. This event will be called when the Tag's "dismiss" button is activated.
+
+```jsx
+const onDismiss = (event) => {
+  window.alert('Goodbye Tag! 👋');
+};
+
+<React.Fragment>
+  <Tag color="#cd2f83" onDismiss={onDismiss}>Dismiss me!</Tag>
+</React.Fragment>
+```
+When a Tag is removable, it will be indicated with a "dismiss" button (stylised as an `X`/"close" icon).
+
+
+### Placement
+
+Tags should be in close proximity to the data that they're labelling so as to clearly identify and reinforce the relationship.
+
+## Voice & tone
+
+- Tag labels should use sentence case.
+- Tag labels should be short, succinct, and easily scannable. Avoid tags that are longer than three words.
+- If you need more than one word in a tag label, separate the words with a space.
+
+## Accessibility
+
+### Typography and type case
+
+Nautilus defaults to showing tag labels in small-caps to distinguish them from body copy and interactive elements.
+
+Small caps can be difficult for users to parse, so it's important to use them sparingly. Remember that Tags are intended only to present *supplementary* data that isn't critical to understanding the core content. Tags should *never* contain content or data that's essential to understanding the content you're presenting.
+
+Use sentence case when passing text strings to a Tag. Small caps styling will be added by CSS. If you pass a word in all caps, screen readers can behave unpredictably. **Only** pass labels in all caps if they're abbreviations and should be read out loud as such.
+
+### Grouping Tags
+
+When showing a group or list of tags, use a Group or List component to indicate that tags are part of the same grouping.
```

Comment:
Do we plan on providing a `Group` component? It seems pretty low-level and I don't like that kind of "developer-knowledge-required" accessibility approach. I'd rather we mandate a `<Tags.Tag>`/`<Tags.Item>` be a child of `<Tags>` and do it automatically for the output. If a user supplies only a single tag we could modify the markup accordingly and omit the grouping—and we could be smart with an `aria-describedby` element instead of prefixing the label on every tag. But this approach requires the developers work harder or rememeber when to group tags. I don't understand when multiple tags like this *wouldn't* be related. Or if they are, I'd rather *that* be the explicit option someone has to choose.

I feel like we can be better-by-default here.

Reply:
I left responses in Notion but maybe the notifications aren't going through. Let me copy here!

> I'm not sure that makes sense in all cases though—what if we just want a single Tag, for a date or a status maybe?

> Maybe we should have an OPTIONAL <TagGroup> or <Tags> container?

Basically. SOME Tags will warrant a group, but others will not. We need to allow for some way of using both. <Tags.Tag> feels really awkward and fragile when you only need a single Tag. Maybe it would make sense instead to have an optional TagGroup or TagList component?

Reply:
Oops, nope, they are. I just haven't looked. I'll review there.